static/app/views/replays/replaySearchBar.tsx

```diff
@@ -38,10 +50,39 @@ type Props = React.ComponentProps<typeof SmartSearchBar> & {
 };
 
 function ReplaySearchBar(props: Props) {
+  const {organization, pageFilters} = props;
+  const api = useApi();
+  const projectIdStrings = pageFilters.projects?.map(String);
+
+  const getTagValues = useCallback(
+    (tag: Tag, searchQuery: string, _params: object): Promise<string[]> => {
+      if (isAggregateField(tag.key)) {
+        // We can't really auto suggest values for aggregate fields
+        // or measurements, so we simply don't
+        return Promise.resolve([]);
+      }
+
+      return fetchTagValues({
+        api,
+        orgSlug: organization.slug,
+        tagKey: tag.key,
+        search: searchQuery,
+        projectIds: projectIdStrings,
+        includeReplays: true,
```

Comment:
missed this little guy the first time around:

```suggestion
        includeReplays: true,
```

Reply:
yay!